tests/integration/test_analysisd/test_eps/conftest.py

```diff
@@ -0,0 +1,159 @@
+
+    yield
+
+
+def delete_folder_content(folder):
+    """Delete alerts folder content execution"""
+    for filename in os.listdir(folder):
+        filepath = os.path.join(folder, filename)
+        try:
+            shutil.rmtree(filepath)
+        except OSError:
+            os.remove(filepath)
+
+
+@pytest.fixture(scope='function')
+def delete_alerts_folder():
+    """Delete alerts folder content before and after execution"""
+
+    delete_folder_content(ALERT_PATH)
+
+    yield
+
+    delete_folder_content(ALERT_PATH)
+
+
+def get_wazuh_internal_options() -> List[str]:
+    """Get current `internal_options.conf` file content.
+
+    Returns
+        List of str: A list containing all the lines of the `ossec.conf` file.
+    """
+    with open(WAZUH_INTERNAL_OPTIONS) as f:
+        lines = f.readlines()
+    return lines
+
+
+def set_wazuh_internal_options(wazuh_local_internal_options: List[str]):
+    """Set up Wazuh `local_internal_options.conf` file content.
+
+    Returns
+        List of str: A list containing all the lines of the `local_interal_options.conf` file.
+    """
+    with open(WAZUH_INTERNAL_OPTIONS, 'w') as f:
+        f.writelines(wazuh_local_internal_options)
+
+
+def change_internal_options(param, value, value_regex='[0-9]*'):
+    """Change the value of a given parameter in internal_options.
+
+    Args:
+        param (str): parameter to change.
+        value (obj): new value.
+        value_regex (str, optional): regex to match value in local_internal_options.conf. Default '[0-9]*'
+    """
+    add_pattern = True
+    with open(WAZUH_INTERNAL_OPTIONS, "r") as sources:
+        lines = sources.readlines()
+
+    with open(WAZUH_INTERNAL_OPTIONS, "w") as sources:
+        for line in lines:
+            sources.write(
+                re.sub(f'{param}={value_regex}', f'{param}={value}', line))
+            if param in line:
+                add_pattern = False
+
+    if add_pattern:
+        with open(WAZUH_INTERNAL_OPTIONS, "a") as sources:
+            sources.write(f'\n\n{param}={value}')
+
+
+@pytest.fixture(scope='session')
+def configure_internal_options_eps():
+    """Fixture to configure the internal options file."""
+
+    # Backup the old local internal options
+    backup_internal_options = get_wazuh_internal_options()
+
+    change_internal_options('analysisd.event_threads', '1')
+    change_internal_options('analysisd.syscheck_threads', '1')
+    change_internal_options('analysisd.syscollector_threads', '1')
+    change_internal_options('analysisd.rootcheck_threads', '1')
+    change_internal_options('analysisd.sca_threads', '1')
+    change_internal_options('analysisd.hostinfo_threads', '1')
+    change_internal_options('analysisd.winevt_threads', '1')
+    change_internal_options('analysisd.rule_matching_threads', '1')
+    change_internal_options('analysisd.dbsync_threads', '1')
+    change_internal_options('remoted.worker_pool', '1')
```

Comment:
Make `change_internal_options` accepts a map instead in order to avoid multiple function calls.

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3181/commits/f817116435639fa720611bd68eb90ced7c606062)